adb_graphics/default_specs.yml

```diff
@@ -55,6 +55,31 @@
 #      anchor needs to be applied before another section.
 #
 
+1hsm: # Helicity
```

Comment:
How does this helicity differ from the hlcy section? There is already a sr01 level in that section.

Reply:
> Wow! Another LARGE set down!
> 
> I noticed that all the helicity plots needed the wind barbs turned on.

Some of them wind shear and others plot storm motion, all as wind barbs.  It should be easy to turn that on if we can distinguish which winds to plot.

Reply:
These are the same, I was going by the spreadsheet name and didn't notice the placeholders under hlcy.  I can move them if we prefer to have them under hlcy.